modules/sdk-coin-ada/package.json

```diff
@@ -43,11 +43,11 @@
   "dependencies": {
     "@bitgo/sdk-core": "^1.0.1",
     "@bitgo/statics": "^6.17.0",
-    "@emurgo/cardano-serialization-lib-browser": "^10.2.0",
```

Comment:
this is used by webpack for module substitution, we're going to need to add an exception for this since the nodejs version will not work in the browser.

Reply:
I'll add that in, thanks!